pennylane_lightning/lightning_tensor/_measurements.py

```diff
@@ -74,7 +80,11 @@ def _measurement_dtype(self):
 
         Returns: the Measurements class
         """
-        return MeasurementsC64 if self.dtype == np.complex64 else MeasurementsC128
+        # return MeasurementsC64 if self.dtype == np.complex64 else MeasurementsC128
+        if self._method == "tn":
```

Comment:
"exact"?

Reply:
We changed the method name to follow the same options as `default.tensor`